app/db.py

```diff
@@ -60,63 +63,76 @@ class OHLCV(Base):
 
 def symbol_exists(symbol):
     """ check if the stock symbol exists in the database """
-    return session.query(exists().where(OHLCV.symbol==symbol))[0]
+    with session_scope() as session:
+        return session.query(
+            exists()
+            .where(
+                OHLCV.symbol==symbol,
+            )
+        )[0][0]
```

Comment:
I started looking at `symbol_exists` because we talked about the query result not being ideal (especially needing to use `[0][0]`), but I can't find any uses of the function other than in the tests.  Is it used in the application anywhere?  If not, it (and it's tests) should be removed.